RadioactiveRod.gd

```diff
@@ -0,0 +1,29 @@
+extends StaticBody
+
+
+var radiation_array = []
+var center_x = 10.5
+var center_y = 10.5
+var center_z = 10.5
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	var csv_file = File.new()
+	csv_file.open("res://points.csv", File.READ)
+	csv_file.get_csv_line() # skip past first line of labels
+	
+	# copy all values of the csv file into an array
+	while !csv_file.eof_reached(): 
+		var csv_line = csv_file.get_csv_line()
+		var block = []
+		for i in range(0, csv_line.size()):
+			block.append(float(csv_line[i]))
+		radiation_array.append(block)
```

Comment:
The way you are currently using/querying this data, don't you just need the first value from each row?

Reply:
Yes, but in the future where we have radiation maps that don't follow the assumptions I've made (each voxel is a 1x1x1 cube, the radiation map is sorted in such a way that fits how I find the indices of voxels), I will need the bounds of the voxels to check if the player's position is inside the voxel.